### PR TITLE
fix: whois.nic.me parsing

### PR DESCRIPTION
**Pull request by raszi**
Closes #33

### CODE REVIEW

**Comment by raszi** [comment]
So I expect this to fail again in Travis. :(

**Comment by raszi** [comment]
@weppos any chance to merge this?

File: whois-parser.gemspec [hunk 1]
```diff
@@ -11,7 +11,288 @@ Gem::Specification.new do |s|
...
+  s.files = [
+    ".yardopts".freeze,
```
Comment (raszi) [inline whois-parser.gemspec]: I did this change to help the future PRs and merges.

Comment (weppos) [inline whois-parser.gemspec]: @raszi the gemspec is generated automatically, any change here will be overridden.

Comment (raszi) [inline whois-parser.gemspec]: Ohh, shoot, I did not know that. How can I update that?

Comment (raszi) [inline whois-parser.gemspec]: `rake gemspec`?

**Comment by weppos** [comment]
@raszi I apologize, I've been really busy with the 4.0 release. I'll review this asap.

**Comment by njh** [comment]
I have just tested this branch and it is working well for me - it gets the main information I care about without failing for [njh.me](http://njh.me/).

However it doesn't seem to parse the registrar field (returns nil for my domain).

File: lib/whois/parsers/whois.nic.me.rb [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
 
 require_relative 'base_afilias'
+require 'whois/scanners/whois.nic.me.rb'
```
Comment (weppos) [inline lib/whois/parsers/whois.nic.me.rb]: Any reason to not use a `BaseIcannCompliant` parser?

Comment (raszi) [reply lib/whois/parsers/whois.nic.me.rb]: Sorry, but I cannot remember, this was one and a half year ago and I am not using this gem anymore.

**Comment by weppos** [comment]
Ok, thanks for the followup.

**Comment by jarthod** [comment]
For the record I ended up starting [my fork](https://github.com/jarthod/whois-parser) in which I fixed all the Afilias based whois parsers (.ME, .ORG, .INFO, .PRO, .BZ, .GI, .LC, .MN, .SC, .VC, .AG, .IN, .AERO, .ASIA, .MOBI, .POST, .XXX) in e2b84e73f43ef692e834c2d151f5c7e50b384eae. So people who need this can use my fork:

```ruby
gem 'whois-parser', github: 'jarthod/whois-parser' # Unofficial but more up-to-date fork, check status at https://github.com/jarthod/whois-parser
```

**Pull request closed without merging**